crates/ruff_linter/src/rules/pylint/snapshots/ruff_linter__rules__pylint__tests__PLW0101_unreachable.py.snap

```diff
@@ -1,4 +1,34 @@
 ---
 source: crates/ruff_linter/src/rules/pylint/mod.rs
 ---
+unreachable.py:18:5: PLW0101 Unreachable code in `after_return`
+   |
+16 |   def after_return():
+17 |       return 1
+18 | /     print("unreachable")
```

Comment:
We need multiline span diagnostics so that we can highlight the "why". (CC: @ntBre might be a good first rule to port)

With our current noqa system. Would a user have to put the noqa on the last line? Would that be confusing if they have

```py
return 1

if a:
	pass
else:
	other # noqa: unreachable
```